scripts/common.bat

```diff
@@ -32,14 +32,20 @@ exit /b %errorlevel%
   
   cd %PROJECT%
   
-  if not [%NO_CLEAN%] == [true] (
+  if defined NO_CLEAN (
```

Comment:
This fails the CI builds, as the release builds no longer get cleaned due to this:
https://github.com/gnustep/tools-windows-msvc/blob/18b758551a0821e1ed586bb318ce2930bbe472fd/build.bat#L164-L165

With this change, we’d need to unset NO_CLEAN above instead.

Reply:
Thanks! I'll fix this.